src/main/java/MainPage/MainPage.java

```diff
@@ -0,0 +1,49 @@
+package MainPage;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import pages.SignInPage;
+import pages.TshirtPage;
+
+public class MainPage {
+  static WebDriver driver;
```

Comment:
Why static?

Reply:
because of this 
![image](https://user-images.githubusercontent.com/82042858/169689323-dc5eaed8-1d54-430a-8f11-bce1d8950563.png)

Reply:
ok